src/System.Numerics.Tensors/System/Numerics/ArrayTensorExtensions.cs

```diff
@@ -6,144 +6,141 @@ namespace System.Numerics
-        /// <param name="array"></param>
-        /// <param name="reverseStride"></param>
-        /// <returns></returns>
-        public static DenseTensor<T> ToTensor<T>(this T[] array, bool reverseStride = false)
+        /// <typeparam name="T">Type contained in the array to copy to the DenseTensor&lt;T&gt;.</typeparam>
+        /// <param name="array">The array to create a DenseTensor&lt;T&gt; from.</param>
+        /// <returns>A 1-dimensional DenseTensor&lt;T&gt; with the same length and content as <paramref name="array"/>.</returns>
+        public static DenseTensor<T> ToTensor<T>(this T[] array)
         {
-            return new DenseTensor<T>(array, reverseStride);
+            return new DenseTensor<T>(array);
         }
 
         /// <summary>
-        /// Creates a copy of this two-dimensional array as a DenseTensor<T>
+        /// Creates a copy of this two-dimensional array as a DenseTensorr&lt;T&gt;
         /// </summary>
-        /// <typeparam name="T"></typeparam>
-        /// <param name="array"></param>
-        /// <param name="reverseStride"></param>
-        /// <returns></returns>
+        /// <typeparam name="T">Type contained in the array to copy to the DenseTensor&lt;T&gt;.</typeparam>
+        /// <param name="array">The array to create a DenseTensor&lt;T&gt; from.</param>
+        /// <param name="reverseStride">False (default) to indicate that the first dimension is most major (furthest apart) and the last dimension is most minor (closest together): row-major.  True to indicate that the last dimension is most major (furtheset apart) and the first dimension is most minor (closest together): column-major.</param>
+        /// <returns>A 2-dimensional DenseTensor&lt;T&gt; with the same dimensions and content as <paramref name="array"/>.</returns>
         public static DenseTensor<T> ToTensor<T>(this T[,] array, bool reverseStride = false)
         {
             return new DenseTensor<T>(array, reverseStride);
         }
 
         /// <summary>
-        /// Creates a copy of this three-dimensional array as a DenseTensor<T>
+        /// Creates a copy of this three-dimensional array as a DenseTensorr&lt;T&gt;
         /// </summary>
-        /// <typeparam name="T"></typeparam>
-        /// <param name="array"></param>
-        /// <param name="reverseStride"></param>
-        /// <returns></returns>
+        /// <typeparam name="T">Type contained in the array to copy to the DenseTensor&lt;T&gt;.</typeparam>
+        /// <param name="array">The array to create a DenseTensor&lt;T&gt; from.</param>
+        /// <param name="reverseStride">False (default) to indicate that the first dimension is most major (furthest apart) and the last dimension is most minor (closest together): akin to row-major in a rank-2 tensor.  True to indicate that the last dimension is most major (furtheset apart) and the first dimension is most minor (closest together): akin to column-major in a rank-2 tensor.</param>
```

Comment:
type-o `furtheset`.  😉

Reply:
I think `major (furthest apart)` and `minor (closest together)` are backwards, aren't they?

From wikipedia:  `In a row-major order, the consecutive elements of a row reside next to each other`

Reply:
No.  The rows themselves are furthest apart, the columns (AKA: consecutive elements of a row) are closest.

Reply:
But the column itself is spread out - it isn't contiguous.  The row is the one that is contiguous (close together).

Reply:
I guess I see what you are saying.  It's really hard to explain these things in a single sentence.